### PR TITLE
Update catalog revision, callGraph and report endpoints

### PR DESCRIPTION
**Pull request by marounkoussaifi**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -4,5 +4,5 @@
   <component name="FrameworkDetectionExcludesConfiguration">
     <file type="web" url="file://$PROJECT_DIR$" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK" />
```
Comment (fviale) [inline .idea/misc.xml:7]: a bit weird that this file is in the repository

Comment (marounkoussaifi) [inline .idea/misc.xml:7]: I just ran the swagger client CLI command as usual. Do you have any suggestions on how to remove it or should I keep it like this?

Comment (fviale) [reply .idea/misc.xml:7]: it's ok actually, I didn't see that it was the catalog-client

**Review by tao-xinxiu — APPROVED** [review]

**Review by fviale — APPROVED** [review]

**Comment by fviale** [comment]
Retest this please

**Pull request merged**